engine/image-loader/tga.lisp

```diff
@@ -0,0 +1,26 @@
+(defpackage lbge.image-loader.tga
+  (:use :cl :lbge.image-loader)
+  (:export tga))
+(in-package lbge.image-loader.tga)
+
+(defun tga (path)
+  "Returns the `lbge.image-loader:image` object for `path` file."
+  (let ((raw (alexandria:read-file-into-string path))
```

Comment:
If the data type is changed to vector as mentioned above, this would have to be changed to `alexandria:read-file-into-byte-vector`